brix11/lib/brix11/system.rb

```diff
@@ -24,6 +24,8 @@ def self.get_cpu_cores
       unless @cpu_cores
         if mswin?
           @cpu_cores = (ENV['NUMBER_OF_PROCESSORS'] || 1).to_i
+        elsif File.exist?('/usr/sbin/sysctl')
+          @cpu_cores = (`/usr/sbin/sysctl -n hw.ncpu` rescue '1').strip.to_i
```

Comment:
`/usr/sbin/sysctl` does also exist on my OpenSuSE linux system, but fails with `sysctl: cannot stat /proc/sys/hw/ncpu: No such file or directory`. This change breaks my Linux multi cpu support, maybe check first if you are on a Mac and second if the sysctl exists?

Reply:
It also fails inside a docker linux build image!

root@45565f35112e:/axcioma# irb
```ruby
irb(main):001:0> (`/usr/sbin/sysctl -n hw.ncpux` rescue 1).strip.to_i
sysctl: cannot stat /proc/sys/hw/ncpux: No such file or directory
=> 0
irb(main):002:0> 
```

This `rescue 1` is obscure, or not?